daemon/src/lib.rs

```diff
@@ -261,6 +263,7 @@ where
             monitor_addr.clone(),
             oracle_addr,
             n_payouts,
+            maker_identity,
```

Comment:
nit: `maker_id` should be sufficient? (it's called `taker_id` in the other actor already...)

Reply:
That's not aligned throughout the codebase. I'd say we do that in a followup.